resources/units/aircraft/OH-6A.yaml

```diff
@@ -0,0 +1,18 @@
+class: Helicopter
+carrier_capable: false
+description:
+  The Hughes OH-6 Cayuse is a single-engine light helicopter designed and
+  produced by the American aerospace company Hughes Helicopters.
+introduced: 1966
+lha_capable: false
+can_carry_crates: false
+manufacturer: Hughes Helicopters
+origin: USA
+price: 3
+role: Light Observation/Light Attack
+variants:
+  OH-6A Cayuse: {}
+tasks:
+  BAI: 200
+  CAS: 200
+  OCA/Aircraft: 200
```

Comment:
Does the Recon task have the ability to use AttackGroup & SeachAndAttack waypoint tasks? Otherwise this might not work...

Reply:
Also, if it actually has these tasks, then we need to add Recon accordingly in `aircraftbehavior.py`

Reply:
I'm not sure if that is relevant in the case of this mod, since I saw somewhere (a reference which I can't find right now) that the OH-6 might not work on AI hands at all, or might work poorly...

Reply:
Well then we definitely need to add the fallbacks either way because even for player flights I believe it will run into an exception during generation of a CAS/BAI/OCA mission.

Reply:
Just checked in DCS, seems like Reconnaissance doesn't have any waypoint-task to attack, except strafing but that won't help us I'm afraid...

I would replace the tasks with Transport only, although I guess it's somewhat unrealistic because I can't imagine the little bird hauling tanks, but that's about the only auto-generated mission I would allow 😅

At the same time we would _need_ these tasks because otherwise you can't plan those flights manually for players either, but I would opt for something that checks whether it's an OH-6 & it's a player flight, inject the Reconnaissance task for `configure_cas` & `configure_oca_strike`